auth/api/iam/generated.go

```diff
@@ -423,6 +426,2603 @@ func (a TokenIntrospectionResponse) MarshalJSON() ([]byte, error) {
 	return json.Marshal(object)
 }
 
+// RequestEditorFn  is the function signature for the RequestEditor callback function
```

Comment:
These additional 2600 LoC are used for a single call in a test. Generate code to a test package so we can give that part of the code another status as *production* code

Reply:
Moved to `e2e` folder.